nawab.py

```diff
@@ -57,11 +77,11 @@ def main():
 
     #Initiate twitter bot
     thread = threading.Thread(target=wrapper, args=(
-        twitter_bot_run, (data, default_dir, ), res))
+        twitter_bot_run, (data, default_dir,level, ), res))
     thread.start()
     while thread.is_alive:
         #Initiate telegram bot
-        tg_bot_run(data, default_dir)
+        tg_bot_run(data, default_dir,level,retweet, manual_tweet)
```

Comment:
You have to check at first if the argument is passed or not by the user. Then only there is a point in passing the argument forward with the subsequent functions.

Reply:
ok I will make the necessary changes.